js/web/src/components/AllReleases.js

```diff
@@ -16,7 +16,7 @@ const Releases = () => {
   const [listView, setListView] = useState(false)
 
   useEffect(() => {
-    getReleasesAll()
+    getReleasesAll(releases.length)
```

Comment:
don't see this argument passed to the function declaration, seems like the function gets length/offset from state and doesn't need it

Reply:
good catch ty